translations/game_modes.tex/fr.po

```diff
@@ -91,6 +97,11 @@ msgid ""
 "Further rules changes are detailed in the Campaign Mission Books.\n"
 "\n"
 msgstr ""
+"\\subsection*{Campagne}\n"
+"Un Mode Solo aux Scénarios interconnectés contre une IA ennemie.\n"
+"Pour les règles spécifiques au Mode Solo\\index{Solo Mode}, voir \\hyperlink{AIrules}{Règles IA}.\n"
```

Comment:
```suggestion
"Pour les règles spécifiques au Mode Solo\\index{Mode Solo}, voir \\hyperlink{AIrules}{Règles IA}.\n"
```
Or maybe like that?

Reply:
We were taking the route of keeping indexes the  same as in english as they are not visible

Reply:
What do you mean by that? If you download the printable version from readme, the index is there.

Reply:
You want to have a French version with English index at the end?

Reply:
Well I didn't know

Reply:
That's OK, that's what I'm here for 😆